### PR TITLE
Add GetAssetInfo acceptance test

### PR DESCRIPTION
**Pull request by l4l**
### Description of the Change

This pr introduce integration tests for GetAssetInfo query

### Benefits

More coverage

### CODE REVIEW

File: test/integration/acceptance/get_asset_info_test.cpp [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      makeQuery(""),
+      checkQueryErrorResponse<
+          shared_model::interface::StatelessFailedErrorResponse>());
+}
+
+/**
+ * C366 Pass a non-existing asset id
+ * @given a user with all required permissions
+ * @when GetAssetInfo is queried on the user
+ * @then query is stateful invalid response
```
Comment (Akvinikym) [inline test/integration/acceptance/get_asset_info_test.cpp]: query **has** stateful... ?

File: test/integration/acceptance/get_asset_info_test.cpp [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  prepareState().sendQuery(
+      makeQuery("inexistent#" + kDomain),
+      checkQueryErrorResponse<shared_model::interface::NoAssetErrorResponse>());
+}
+
+/**
+ * C364 Get asset info without CanReadAssets permission
+ * @given a user without any query-related permission
+ * @when GetAssetInfo is queried on the user
+ * @then query is stateful invalid response
```
Comment (Akvinikym) [inline test/integration/acceptance/get_asset_info_test.cpp]: query **has** stateful... ?

File: test/integration/acceptance/get_asset_info_test.cpp [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+ */
+TEST_F(GetAssetInfo, NoPermission) {
+  prepareState({}).sendQuery(
+      makeQuery(),
+      checkQueryErrorResponse<
+          shared_model::interface::StatefulFailedErrorResponse>());
+}
```
Comment (Akvinikym) [inline test/integration/acceptance/get_asset_info_test.cpp]: Isn't it useful to have also test for sending `GetAssetInfo` from non existent user? I think, it's covered in other tests, but still

File: test/integration/acceptance/get_asset_info_test.cpp [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  /**
+   * Creates valid GetAssetInfo query of the current asset
+   * @return built query
+   */
+  auto makeQuery() {
+    return makeQuery(kAssetId);
+  }
+
+  /**
+   * @param asset is asset name for checking
```
[the remark this answers is not in the document]
Comment (l4l) [reply test/integration/acceptance/get_asset_info_test.cpp]: I think line with `@return` is enough

File: test/integration/acceptance/get_asset_info_test.cpp (hunk 3 above)
Comment (l4l) [reply test/integration/acceptance/get_asset_info_test.cpp]: The task has no such test. But indeed that might be useful so I'll add it

File: test/integration/acceptance/get_asset_info_test.cpp [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+/**
+ * C366 Pass a non-existing asset id
+ * @given a user with all required permissions
+ * @when GetAssetInfo is queried on the user
+ * @then query is stateful invalid response
+ */
+TEST_F(GetAssetInfo, NonexistentAsset) {
+  prepareState().sendQuery(
+      makeQuery("inexistent#" + kDomain),
```
[the remark this answers is not in the document]
Comment (l4l) [reply test/integration/acceptance/get_asset_info_test.cpp]: Google says that both are acceptable
So I use both in favor of diversity 🌈

**Pull request merged**